CHANGES.rst

```diff
@@ -295,6 +295,9 @@ astropy.io.votable
 astropy.modeling
 ^^^^^^^^^^^^^^^^
 
+- Fix the shape of the outputs when a model set is evaluated with
+  ``model_set_axis`` different from the one it was initialized with. [#7317]
```

Comment:
Sorry to be a bit pedantic, but I think this description could be a bit more specific, ie. it's when evaluating with `False` rather than just with any value that differs from initialization (right?).

Reply:
Good point. Done.